src/Talis/Persona/Client/Base.php

```diff
@@ -172,33 +174,42 @@ protected function getLogger()
     }
 
     /**
+     * Create a http client
+     * @param boolean $skipRevalidation whether to skip the validation of the
+     *    remote etag/cache headers to ensure that the cached response is still
+     *    valid, or trust the originally set expiry time
      * @return \Guzzle\Http\Client
      */
-    protected function getHTTPClient()
+    protected function getHTTPClient($skipRevalidation = false)
     {
-        if ($this->httpClient === null) {
```

Comment:
I don't believe that there is a reason to cache the http client. The addition of the revalidate attribute being set within the CachePlugin makes this caching tricky to implement. For now I would rather remove the caching as I cannot define why we need it.

Reply:
Is it now safe to remove https://github.com/talis/talis-php/pull/9/files#diff-9b4a015d45cffb1051f161a555370932R45 ?